astro/src/content/docs/apis/identity/verify.mdx

```diff
@@ -0,0 +1,173 @@
+
+## Start Identity Verification
+
+This API allows you to generate a verification code for a User's identity. This code can be sent to the User via email or SMS using the [Send Identity Verification API](#send-identity-verification). The User will then use this code to complete the verification process.
+
+### Request
+
+<API method="POST" uri="/api/identity/verify/start" authentication={["api-key"]}
+     title="Start identity verification" />
+
+<XFusionauthTenantIdHeaderScopedOperation />
+
+#### Request Body
+
+<APIBlock>
+  <APIField name="loginId" type="String" required>
+    The login identifier of the User identity to begin verification for. The login identifier can be one of the following:
+    * Email address
+    * Phone number
+
+    <Aside type="note">
+      For <InlineField>email</InlineField>, the value will be used exactly as provided to resolve a User. If the provided value contains leading or trailing white space, these characters will not be removed, and then would only match if the values stored in FusionAuth also contain these same leading or trailing spaces.
+
+      For <InlineField>phoneNumber</InlineField>, <InlineField>loginId</InlineField> can be any valid phone number (in E.164, national, or international format) and FusionAuth will canonicalize the value. Thus the User will be resolved if the User's E.164 formatted phone number matches the canonical E.164 phone number from <InlineField>loginId</InlineField>. If no country code is provided, a US/NANP country code of 1 is assumed.
+    </Aside>
+  </APIField>
+  <APIField name="loginIdType" type="String" required>
+    The identity type that FusionAuth will compare the <InlineField>loginId</InlineField> to. Can be one of the following:
+
+    * `email`
+    * `phoneNumber`
+  </APIField>
+
+  <APIField name="applicationId" type="UUID" optional>
+    An application Id. If this is not provided and there are multiple tenants, the <InlineField>X-FusionAuth-TenantId</InlineField> header is required.
+  </APIField>
+  <APIField name="state" type="Object" optional>
+    An optional object that will be returned un-modified when you complete verification. This may be useful to return the User to particular state once verification is completed.
+  </APIField>
+  <APIField name="verificationStrategy" type="String" optional defaults="tenant value">
+    The strategy to use when verifying the User. Can be one of the following:
+
+    * `ClickableLink` - send the User a code with a clickable link.
+    * `FormField` - send the User a short code intended to be manually entered into a form field
+
+    The default, when `loginId` is an email address, is <InlineField>tenant.emailConfiguration.verificationStrategy</InlineField>. When `loginId` is a phone number, the default is <InlineField>tenant.phoneConfiguration.verificationStrategy</InlineField>.
+  </APIField>
+</APIBlock>
+
+### Response
+
+<StandardPostResponseCodes never_search_error />
```

Comment:
We don't get into the details of errors here (template not found, messenger, etc.). I believe that is customary/OK.